### ISSUE TITLE
Support for CalDAV

### ISSUE THREAD

**Issue by TheZoker** [chrisvel/tududi#1]
Would be amazing if this app supports caldav synchronisation. This would allow to use this as a webfrontend for a caldav server (e.g. etesync or Baikal etc.) and to use it with the amazing tasks.org android/iOS app. 

I currently use the app and I'm missing a nice web interface to also work on my tasks. Tu Du Di looks amazing and would perfectly fit that requirement (if a docker image is available).

Would appriciate to hear some feedback on this idea! 

Bonus point: You instantly have a app to work alongside your project :D

**Comment by skyrunner15** [comment]
+1 so I can use it on my Android phone like DavX+Tasks.org

**Comment by chrisvel** [comment]
@TheZoker Can you please explain in a sentence how could this feature help with personal task management ? The reason I am asking is because I have no experience with CalDAV and I want to evaluate If this would be helpful for me and for other people too.

**Comment by TheZoker** [comment]
@chrisvel So the main idea would be, to be able to manage task from the tududi and from a mobile app. And since CalDAV is a open standard, there are already many apps that support this (e.g. tasks.org, DavX etc.)
With a CalDAV integration I can add tasks from my mobile app and they will popup in the webapp. When I mark a task as done in the webapp it will also be marked as done in the mobile app. And so on. 
The main benefit is, that I can manage the tasks no matter which device I use. I can add tasks on the go and check them when I'm working on my notebook.
Does this make sense? Let me know if you need more information! 😄

**Comment by chrisvel** [comment]
@TheZoker yeah, I know in generic terms what CalDAV is and I have used DavX in the past but I haven't developed anything related to it ever. I guess it could be easy to add an integration - the difficult part would be to make it compatible with the models and the schema. I promise to research it though and come up in a few days with some valuable results. Thanks a lot!

**Comment by groetg** [comment]
+1 for me too.
